ArmaforcesMissionBot/Modules/Signups.cs

```diff
@@ -281,6 +281,15 @@ public async Task AddTeam([Remainder]string teamText)
                             team.Pattern += $"{slot.Emoji} [{slot.Count}] {slot.Name} ";
                         }
                     }
+                    
+                    foreach (var slotEmoji in team.Slots)
+                    {
+                        if (team.Slots.FindAll(x => x.Emoji == slotEmoji.Emoji).Count > 1)
+                        {
+                            await ReplyAsync("Zdublowałeś reakcje. Poprawiaj to!");
+                            return;
+                        }
+                    }
```

Comment:
You could do better, without using foreach loop, with reduced nesting.
```suggestion
                    if (team.Slots
                        .GroupBy(x => x.Emoji)
                        .Any(x => x.Count() > 1))
                    {
                        await ReplyAsync("Zdublowałeś reakcje. Poprawiaj to!");
                        return;
                    }
```